### DISCUSSION TITLE
Course certificate generation at end of completion

### DISCUSSION THREAD

**Discussion by ArjunSharda** [learnhouse/learnhouse#388] [Ideas]
Hey there!

I was looking at Learnhouse (which is so amazing, by the ways) and was just curious if there are any intentions to add certificate generation which can be automatically given to users at the end of a course completion. This would be great for universities & organizations. Just a thought out there or has this already been implemented?

**Comment by swve** [comment]
Hey sorry for replying (so) latem this sis something planned as part of the "Learning Paths" feature

**Comment by swve** [comment]
This has been implemented @ArjunSharda
